examples/subplots_axes_and_figures/subplot.py

```diff
@@ -8,6 +8,35 @@
 import numpy as np
 import matplotlib.pyplot as plt
 
+###############################################################################
+
+x1 = np.linspace(0.0, 5.0)
+x2 = np.linspace(0.0, 2.0)
+
+y1 = np.cos(2 * np.pi * x1) * np.exp(-x1)
+y2 = np.cos(2 * np.pi * x2)
+
+fig, (ax1, ax2) = plt.subplots(2, 1)
+fig.suptitle('A tale of 2 subplots')
+
+ax1.plot(x1, y1, 'o-')
+ax1.set_ylabel('Damped oscillation')
+
+ax2.plot(x2, y2, '.-')
+ax2.set_xlabel('time (s)')
+ax2.set_ylabel('Undamped')
+
+plt.show()
+
+#############################################################################
+#
+#
+# Alternative Method For Creating Multiple Plots
+# """"""""""
+#
+# Subplots can also be generated using `subplot()` as in the following example:
```

Comment:
```suggestion
# Subplots can also be generated using `~.pyplot.subplot()` as in the following example:
```

To remove ambiguity. Will need line-wrapping to stay below 80 char limit.

Reply:
Hi, thanks for the review. I fixed it